site/en/blog/fledge-api/index.md

```diff
@@ -1206,7 +1207,7 @@ FLEDGE grew out of TURTLEDOVE and a collection of related proposals for modifica
    [Product-level](https://github.com/WICG/turtledove/blob/main/PRODUCT_LEVEL.md) TURTLEDOVE
    modifications improved the anonymity model and personalization capabilities of the on-device auction
 -  [PARAKEET](https://github.com/WICG/privacy-preserving-ads/blob/main/Parakeet.md) is
-   Microsoft's proposal for a TURTLEDOVE-like ad service that relies on a trusted proxy server
+   Microsoft's proposal for a TURTLEDOVE-like ad service that relies on a proxy server running in a TEE
```

Comment:
Maybe link TEE to definition?

Reply:
Hmm, the definition is linked in line 1201 and I'm wondering if the links are too close to each other since they will point to the same article. I've added "(TEE)" in line 1201. Let me know if that works.